internal/buildscripts/packaging/tests/package_test.py

```diff
@@ -43,6 +43,15 @@ def get_package(distro, name, path):
         return None
 
 
+def get_rpm_command(container):
```

Comment:
Could this be something like `get_libcap_command` and return the whole thing instead of using an additional zypper check?